app/marsRoverPath/index.js

```diff
@@ -0,0 +1,45 @@
+var dragItem = document.querySelector('#rover-sprite');
+
+var active = false;
+var currentX;
+var currentY;
+var initialX;
+var initialY;
+var xOffset = 0;
+var yOffset = 0;
+
+dragItem.addEventListener('mousedown', dragStart, false);
+dragItem.addEventListener('mouseup', dragEnd, false);
+dragItem.addEventListener('mousemove', drag, false);
+
+function dragStart(event) {
+	initialX = event.clientX - xOffset;
+	initialY = event.clientY - yOffset;
+	if (event.target === dragItem) {
+		active = true;
+	}
+}
+
+function dragEnd() {
+	initialX = currentX;
+	initialY = currentY;
+
+	active = false;
+}
+
+function drag(event) {
+	if (active) {
+		event.preventDefault();
+		currentX = event.clientX - initialX;
+		currentY = event.clientY - initialY;
+
+		xOffset = currentX;
+		yOffset = currentY;
+
+		setTranslate(currentX, currentY, dragItem);
+	}
+}
+
+function setTranslate(xPos, yPos, element) {
+	element.style.transform = 'translate3d(' + xPos + 'px, ' + yPos + 'px, 0)';
```

Comment:
Given that we're not dealing with the third dimension, can probably just use `translate` rather than `translate3d`

Reply:
Done :)